apps/web/src/components/CommunityInterestDialog/DevelopmentProgramInterestItem.tsx

```diff
@@ -172,8 +180,7 @@ const DevelopmentProgramInterestItem = ({
         <span>{label}</span>
         <span
           data-h2-font-size="base(caption)"
-          {...(!developmentProgramInterest?.participationStatus ||
-          !developmentProgramInterest?.completionDate
```

Comment:
Okay, this is the reason why I'm getting the font discrepancy, completion date is required when status is completed 
So I think it should be something like 

```
{...
(!developmentProgramInterest?.participationStatus ||
       (   
       developmentProgramInterest?.participationStatus === COMPLETED &&
       !developmentProgramInterest?.completionDate
       )
```

![image](https://github.com/user-attachments/assets/2ec7d182-fd4f-425a-858c-06409aa3abdb)

Reply:
Thanks for digging in an figuring that out. Should be fixed now after [fix styling conditional logic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13010/commits/d4ad7ae91f7c3cbac963d9e89ba2d8a1ec26ca8c)

Reply:
Thanks, it works now

![image](https://github.com/user-attachments/assets/cf814c0e-bbab-41cc-b3a2-4b1a4727f5c5)